lighthouse/classes/messages/sequencescape_messages.py

```diff
@@ -55,6 +55,7 @@ def send_to_ss(self):
             data=message.payload(),
         )
 
+    # TODO: this is a static method
```

Comment:
I think it's best when TODOs describe what needs to be done.  Otherwise when someone else finds this it's just stating a fact about the code and they don't know whether the TODO is left over where someone was asking that it be made a static method or that it currently is one and it shouldn't be.